plugins/backend/catalog-backend-module-okta/src/providers/OktaGroupEntityProvider.ts

```diff
@@ -85,7 +83,7 @@ export class OktaGroupEntityProvider extends OktaEntityProvider {
     this.logger.info(`Providing group resources from okta: ${this.orgUrl}`);
     const groupResources: GroupEntity[] = [];
 
-    const client = this.getClient(this.orgUrl);
+    const client = this.getClient(this.orgUrl, ['okta.groups.read']);
```

Comment:
Can you move these scopes into a constant, please?

Reply:
Hope you don't mind but I've gone one step further and used a union type which gives us better type safety AND hints for ides to suggest/intellisense :)

Reply:
Not at all!: )